backbone/src/main/java/org/researchstack/backbone/ui/step/body/TextQuestionBody.java

```diff
@@ -70,7 +71,14 @@ public View getBodyView(int viewType, LayoutInflater inflater, ViewGroup parent)
         // Format EditText from TextAnswerFormat
         TextAnswerFormat format = (TextAnswerFormat) step.getAnswerFormat();
 
-        editText.setSingleLine(!format.isMultipleLines());
+        if(format.isMultipleLines()) {
```

Comment:
When I set the answer format to have multiple lines, it was still displayed as a single line input.  Adding these extra parameters allows it to be displayed as a multi line text area.  I am not sure if this change has a negative affect on any existing steps.  If so, I could create a new step with these parameters.   I just set the number of lines to 5 for now but this could also be an additional parameter or something.